operations/productcategories.md

```diff
@@ -0,0 +1,123 @@
+# Product categories
```

Comment:
I've included it in this PR because we we're adding this as part of multi-property and wanted to have first changelog as a whole so you see what was multi-property supported easily.